public/javascripts/data/storyManager.js

```diff
@@ -0,0 +1,45 @@
+const Database = require('better-sqlite3');
+
+exports.displayAllStories = () => {
+    
+    // Get all stories (an array of objects)
+    let db = new Database('public/javascripts/data/stories.db');
+    let allStories = db.prepare('SELECT * FROM stories').all();
```

Comment:
Consider using an `ORDER BY` clause to obtain the most recent stories inserted to the database

Reply:
I was able to do this and use the forEach loop. I think it is much better.

`// Get all stories (an array of objects)
    let db = new Database('public/javascripts/data/stories.db');
    let allStories = db.prepare('SELECT * FROM stories ORDER BY id DESC').all();

    // Get the first five paragraphs (if that many) from each story.
    let i = allStories.length;
    allStories.forEach( (story) => {
        let paraStmt = db.prepare('SELECT * from paragraphs WHERE stories_id = (?) LIMIT 5');
        story['paragraphs'] = paraStmt.all(i);
        i--;
    });
`

Reply:
You won't need the `i` variable - Use the `id` property of the `story` object